clients/admin-ui/tsconfig.json

```diff
@@ -34,9 +34,5 @@
     "**/*.tsx",
     "**/*.d.ts"
   ],
-  "exclude": [
-    "node_modules",
-    "cypress/**/*.ts",
-    "../../admin-ui/cypress.config.ts"
```

Comment:
this `../../admin-ui/cypress.config.ts` file doesn't exist (there's no `../../admin-ui` folder, only `../admin-ui`. so I deleted the line since it doesn't seem like it's doing anything